bciers/apps/administration/tests/components/operators/OperatorForm.test.tsx

```diff
@@ -683,4 +568,134 @@ describe("OperatorForm component", () => {
       ).toBeVisible();
     });
   }, 60000);
+  it("loads existing readonly Operator form data for an internal user", async () => {
```

Comment:
Only change to this test is mocking the session and moving it last (partly to keep the internal/external tests together, partly so I don't have to reset the mock every time)